tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -34,6 +35,7 @@ const AXIS_FONT = '11px Roboto, sans-serif';
   templateUrl: 'line_chart_axis_view.ng.html',
   styleUrls: ['line_chart_axis_view.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Can you add a comment explaining why you need to set this? Maybe even refer to the components we are reaching into and/or the css classes we are referring to?

Reply:
Done